shared/src/domain-logic/licenseStatusHelpers.ts

```diff
@@ -0,0 +1,37 @@
+import { LicenseName, LicenseStatus, LicenseStatusItem, taskIdLicenseNameMapping } from "../license";
+import { Business, TaskProgress } from "../userData";
+
+export const getNonLicenseTasks = (business: Business): Record<string, TaskProgress> =>
+  Object.fromEntries(
+    Object.entries(business.taskProgress).filter(
+      ([key]) => !Object.keys(taskIdLicenseNameMapping).includes(key)
+    )
+  );
```

Comment:
[dust] - We can just take whatever type `taskProgress` is instead of `Business` as an argument to this function since that's the only field we use. It has the slight benefit of decoupling the function from the structure of a `Business`.

Reply:
Hey Lex, we have an established pattern where functions take in the Business as the argument, we typically don't provide specific attributes for the functions. This approach reduces the cognitive load on the devs. I'll leave it as is, but open to discussing during techtro if you think it's worth discussing your suggestion.